documentation/modules/appendix_crds.adoc

```diff
@@ -1446,7 +1446,7 @@ Used in: xref:type-KafkaSpec-{context}[`KafkaSpec`]
 |Configuration of the User Operator.
 |tlsSidecar
 |xref:type-TlsSidecar-{context}[`TlsSidecar`]
-|TLS sidecar configuration.
+|**The `tlsSidecar` property has been deprecated.** TLS sidecar was removed in Strimzi 0.41.0. TLS sidecar configuration.
```

Comment:
Here it FMPOV applies as well -> if the property is deprecated, should we have here that the property is ignored, thus TLS sidecar will not be deployed?

Reply:
Yep, this is auto generated from the annotation.

Reply:
Ah okay, sorry, didn't know that